src/components/auth/mypage/mypage/menu/item-menu.tsx

```
@@ -2,15 +2,35 @@ import * as M from '../../../../../styles/auth/mypage/mypage.style';
       localStorage.clear();
       alert('로그아웃이 완료되었습니다.');
       navigate('/login');
+    } else if (item.name === '회원 탈퇴') {
+      const confirmed = window.confirm('정말로 회원탈퇴를 진행하시겠습니까?');
+      if (!confirmed) return;
+
+      try {
+        const res = await useDeleteMutation({ userId });
+
+        if (res.isSuccess) {
+          localStorage.clear();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)